modules/stylesheet.py

```diff
@@ -0,0 +1,7 @@
+# Colors
+BACKGROUND = (102, 0, 51)
```

Comment:
```suggestion
CERISE =  (102, 0, 51)
BACKGROUND = CERISE
```

This could be added to make it easier to change later. Same for border et.c.